hack/operatorhub/internal/container/container.go

```diff
@@ -342,22 +390,26 @@ func isImageScanned(httpClient *http.Client, apiKey, projectID, tag string) (ima
 	return nil, false, nil
 }
 
+func doPublish(image *Image, c CommonConfig) error {
+	return doOperationForImage(image, c, publishOperation)
+}
```

Comment:
Nit: Not sure of the interest of this minimal function which creates an indirection for not much.